lib/bolt/transport/base.rb

```diff
@@ -68,6 +68,20 @@ def with_events(target, callback)
         result
       end
 
+      def provided_features
+        []
+      end
+
+      def default_input_method
```

Comment:
Out of curiosity, is there a reason we prefer making these methods instead of constants?

Reply:
we want one clear thing to override. methods are better than constants for that.